### ISSUE TITLE
Homepage

### ISSUE THREAD

**Issue by tctcodes** [USM-ACM-IEEE/acmWebsite-React#14]
# ACM Website

Before opening a new issue, please take a moment to review our [**community guidelines**](https://github.com/USM-ACM/acmWebsite/blob/master/.github/CONTRIBUTING.md) to make the contribution process easy and effective for everyone involved.

**Before opening a new issue, you may find an answer in already closed issues**:
https://github.com/USM-ACM/acmWebsite/issues?q=is%3Aissue+is%3Aclosed

## Issue Type

- [ ] [Bug](https://github.com/USM-ACM/acmWebsite/blob/master/.github/CONTRIBUTING.md#bugs)
- [x] [Feature](https://github.com/USM-ACM/acmWebsite/blob/master/.github/CONTRIBUTING.md#features)

## Description
Generating the basic about page.

**Comment by tctcodes** [comment]
@jordanparker32 and @jonathon-evans  Before starting any code, be sure to use Lucidchart to create a wireframe to lay out the look of the homepage.  When that is completed, post a link to your Lucidchart wireframe here and request a review from timthom64.  Thank you!

**Comment by jordanparker32** [comment]
Home Page has been designed in lucidchart under "Landing Page".

**Comment by Quinn-Donnelly** [comment]
Ensure that this file is shared and had been created in the ACM Website folder on lucid chart as it is currently not appearing.

**Comment by jordanparker32** [comment]
The landing page is up

**Comment by tctcodes** [comment]
Thank you! I will look at it and get you feedback either later today, or definitely by Saturday night/Sunday morning.

**Comment by tctcodes** [comment]
Good Work! Thank you for providing the link for the website you modeled your design after.  I think the overall design and layout could look really neat.  

I think the big picture button idea would look really cool.   What are y'all thinking of using for the pictures for the big buttons?.  

I also, like the idea of having a little place for the people who worked on the website somewhere on the about page.  That may even function to help people get more involved.  If they put this project on their resume they can have something to point to.

Where is polling/voting/attendance going to live?

I think it would be cool to have a place to list job postings and upcoming career events, but we shall forever and always receive emails from Ms. Sherry.  

As for news, @Quinn-Donnelly, I know we spoke about using the calendar and Discord as the primary place for disseminating news and upcoming local ACM events.  However, I think having a place to announce upcoming conferences, contests, competitions, seminars/talks, and other computing related events could be cool. Things that aren't ACM mandatory or strongly recommended could possibly be listed here. 

Overall, great work!

**Comment by Quinn-Donnelly** [comment]
@timthom64 I agree I think we can add this functionality to the backend since the first two projects they are assigned are fairly easy to accomplish. Maybe project 3 will be to implement a news system and then the front end can use it.

**Comment by Quinn-Donnelly** [comment]
Also I notice two pages on LucidChart that are a home / landing page if @jordanparker32 or @jonathon-evans could pull a link from their page (make it a view only link, which can be found in top right share button).

**Comment by jordanparker32** [comment]
Why do you want us to do that? To show which design we are going with or because you don’t have the lucid chart space?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Quinn-Donnelly** [comment]
This issue thread is meant to track the progress of tasks so that the maintainers don't need to go looking for information. So just post all links that are related to this task so that in the future if someone needs to reference this issue it has all available information (keeping in mind they may not have access to LucidChart). If they are both your potential designs that is ok just post both.

**Comment by tctcodes** [comment]
@jordanparker32 Please post a link to your mock-up here.

**Comment by jordanparker32** [comment]
http://www.chdmlr.com
